addons/website/static/src/less/website.ui.less

```diff
@@ -12,6 +12,8 @@ body.o_connected_user {
     .o-position-absolute(0, 0, auto, 0);
     position: fixed;
     z-index: 1001;
+    border: none;
```

Comment:
Well, I am not sure. What is the original purpose here? If it is removing the border, I would prefer to remove it in navbar.less and remove all the dirty `- 2`. Backend navbar and frontend navbar are meant to be the same ones.